src/nextcloud/bin/nextcloud-cron

```diff
@@ -11,6 +11,13 @@ done
 printf "done\n"
 
 while true; do
-	run-php "$SNAP/htdocs/cron.php"
-	sleep 15m
+
+  # exit cron loop if interval is "-1"
+	if [ "$(cronjob_interval)" != "-1" ]; then
+	  run-php "$SNAP/htdocs/cron.php"
+	else
+	  break
```

Comment:
I think this whole loop could be simplified into:

```bash
while true; do
    # Exit the entire job if the requested interval is -1
    if [ "$(cronjob_interval)" = "-1" ]; then
        exit 0
    fi

    run-php "$SNAP/htdocs/cron.php"
    sleep "$(cronjob_interval)"
done
```

Reply:
Didn't think about that. This is definitely a better solution.